collections/ansible_collections/purestorage/flasharray/plugins/modules/purefa_volume.py

```diff
@@ -24,6 +24,8 @@
   name:
     description:
     - The name of the volume.
+      Volume could be created in a POD with this syntax: <POD>::<VOLUME_NAME>
```

Comment:
You need to fix your YAML syntax and add a dash to the beginning of each line. 
Notice that you are failing the sanity tests for YAML

Reply:
- Done for the dashes
- I tried ```\:\:``` to see it is passing the sanity tests

Reply:
This ```syntax:``` was causing the issue. I removed the ```:```.  
I accepted the license but it takes time to get updated in the CI.